tests/e2e/utils/flows.js

```diff
@@ -427,6 +428,21 @@ export const merchantWCP = {
 		} );
 	},
 
+	changeTheme: async ( themeSlug ) => {
+		await page.goto( WP_THEMES_PAGE, {
+			waitUntil: 'networkidle0',
+		} );
+
+		if ( await page.$( `.theme[data-slug="${ themeSlug }"]` ) ) {
+			await expect( page ).toClick(
+				`.theme[data-slug="${ themeSlug }"] .theme-actions .button.activate`
+			);
+		}
+		await page.waitForNavigation( {
+			waitUntil: 'networkidle0',
+		} );
+	},
+
```

Comment:
Can be removed in favor of `activateTheme` provided by `@wordpress/e2e-test-utils`

Reply:
Removed in https://github.com/Automattic/woocommerce-payments/pull/8164/commits/8a7bbc244edff7f1723e103975ca10733fb3e23b. the suggestion!